### DISCUSSION TITLE
element plus form validation is failed

### DISCUSSION THREAD

**Discussion by 51fe** [vitest-dev/vitest#2311] [Q&A]
See https://github.com/51fe/element-plus-testing/tree/vitest-vtu or https://github.com/51fe/element-plus-testing/tree/vitest-vtl. Run the test,  there are two tests ('shows required error message without input' and 'shows number error message when input a string') won't pass. When triggering validation on vitest, the element plus form is always valid , so the error messages will be hidden. On the other hand , if we run the tests on jest, these two failed test will pass. I think it's a bug, right?

**Comment by 51fe** [comment]
Have I made myself clear?

**Reply by wangjunxian** [reply]
I have the same problems now. 
it said [formEl.validate is not a function].
or  [Failed to resolve component: el-input]
I'm wondering that have you solved these problems?

**Comment by 51fe** [comment]
After 8 month passed, the issue is still here, finally I used jest instead of vitest.
